package.json

```diff
@@ -79,9 +80,7 @@
     "safe-json-stringify": "^1.0.4",
     "serve-favicon": "^2.4.5",
     "socket.io": "^2.0.4",
-    "solc": "^0.4.18",
     "style-loader": "^0.19.0",
-    "web3": "1.0.0-beta.18",
-    "zeppelin-solidity": "^1.3.0"
+    "web3": "^1.0.0-beta.26"
```

Comment:
don't we get web3 dependency from truffle?

Reply:
I see your point. need to dig a little more.